### DISCUSSION TITLE
Theming tools

### DISCUSSION THREAD

**Discussion by orsenkucher** [primer/github-vscode-theme#307] [Q&A]
Do you know some tools to get token of UI element, like we have with `Inspect Editor Tokens and Scopes`, but for vscode itself?

**Comment by simurai** [comment]
Not sure if there is something built in? 🤔  You can inspect the DOM with the DevTools (Help > Toggle Developer Tools), but that doesn't really tell the "workbench tokens" and might only help in guessing the token names.

I usually just set tokens found in the [reference](https://code.visualstudio.com/api/references/theme-color) to red and then see if I can spot something turning red in the workbench. 😆 🤷
